apps/config.py

```diff
@@ -11,6 +11,20 @@ class Config(object):
 
     basedir = os.path.abspath(os.path.dirname(__file__))
 
+    # Persistência dos CLabs (PoC)
+    CLABS_STATE_PATH = os.getenv("CLABS_STATE_PATH", None) 
```

Comment:
this is likely to be used only for the PoC, so I suggest you move it to the controller definition instead of here.